.circleci/test.sh

```diff
@@ -5,10 +5,10 @@
 set -e
 
 echo "==> Lint code"
-ahoy lint
+ahoy lint || exit 1
```

Comment:
did this not give correct exit code?

Reply:
It didn't seem to be exiting on failure despite `set -e`. I'm currently trying to avoid `ahoy` altogether, though. Will see how we go.